mautic/mautic
# fix for non-functional (un)publish dates on the message form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

The `Publish at (date/time)` and `Unpublish at (date/time)` fields are not working on the messages form.
This is because the original attributes (defining the `data-type` and `class`) are not present as an extra attribute is added in the form.
<img width="942" alt="Screenshot 2023-09-16 at 08 28 57" src="https://github.com/mautic/mautic/assets/3983285/d30f32e4-fc42-4d47-9572-97910ac62633">

This PR fixes this by adding the original attributes.

<img width="939" alt="Screenshot 2023-09-16 at 08 29 56" src="https://github.com/mautic/mautic/assets/3983285/466d17d0-7c12-4c25-b857-1b1745caa53d">


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. create a marketing message and verify you can select a date / time in both fields.
